File: aztec/src/main/kotlin/org/wordpress/aztec/watchers/event/buckets/Bucket.java
package org.wordpress.aztec.watchers.event.buckets;

import org.wordpress.aztec.watchers.event.sequence.UserOperationEvent;

import java.util.ArrayList;

/*
    extend from this class to construct a specific bucket
 */
public abstract class Bucket {
    protected ArrayList<UserOperationEvent> mUserOperations = new ArrayList<>();
    public ArrayList<UserOperationEvent> getUserOperations() {
        return mUserOperations;
    }
}

File: aztec/src/main/kotlin/org/wordpress/aztec/watchers/event/buckets/Bucket.kt
package org.wordpress.aztec.watchers.event.buckets

import org.wordpress.aztec.watchers.event.sequence.UserOperationEvent

import java.util.ArrayList

/*
    extend from this class to construct a specific bucket
 */
abstract class Bucket {
    val userOperations = ArrayList<UserOperationEvent>()
}
